juriscraper/pacer/iquery.py

```diff
@@ -0,0 +1,187 @@
+
+logger = make_default_logger()
+
+
+class iQueryReport(BaseDocketReport, BaseReport):
+    """Parse the iquery.pl ("Query" menu) result.
+
+    This is pretty limited metadata about the case, although it
+    presents some more information for BK cases.
+    """
+
+    CACHE_ATTRS = ['metadata']
+
+    def __init__(self, court_id, pacer_session=None):
+        super(iQueryReport, self).__init__(court_id, pacer_session)
+        # Initialize the empty cache properties
+        self._clear_caches()
+        self._metadata = None
+
+    @property
+    def metadata(self):
+        if self._metadata is not None:
+            return self._metadata
+
+        # The data we're after look like this (respacing):
+        #
+        # <div id="cmecfMainContent">
+        #   <input type="hidden" id="cmecfMainContentScroll" value="0">
+        #   <CENTER>
+        #       <B><FONT SIZE=+1>1:11-cv-10230-MLW</FONT></B>
+        #        Arkansas Teacher Retirement System v. State Street
+        #         Corporation et al
+        #     <BR>
+        #       Mark L. Wolf, presiding
+        #     <BR>
+        #       <B>Date filed:</B> 02/10/2011
+        #     <BR>
+        #       <B>Date terminated:</B> 06/23/2014
+        #     <BR>
+        #       <B>Date of last filing:</B> 06/06/2018
+        #     <BR>
+        #   </CENTER>
+        #
+        # There's a bit more structured data in bankruptcy cases;
+        # but note the extra <B></B> pair in the first line:
+        #
+        # <div id="cmecfMainContent">
+        #   <input type="hidden" id="cmecfMainContentScroll" value="0">
+        #   <CENTER>
+        #       <B><FONT SIZE=+1>18-11572</FONT></B>
+        #       <B></B>Nancy Jean Stevens
+        #     <BR>
+        #       <B>Case type:</B> bk
+        #       <B>Chapter:</B> 7
+        #       <B>Asset:</B> No
+        #       <B>Vol: </B> v
+        #       <b>Judge:</b> Frank J. Bailey
+        #     <BR>
+        #       <B>Date filed:</B> 04/30/2018
+        #       <B>Date of last filing:</B> 06/04/2018
+        #     <BR>
+        #   </CENTER>
+        #
+        # There is some BK variation, both in terms of preence of
+        # docket numbers suffixes, and how the Judge: field name
+        # varies depending on the judge's chief judge status:
+        #
+        # <div id="cmecfMainContent">
+        #   <input type="hidden" id="cmecfMainContentScroll" value="0">
+        #   <CENTER>
+        #       <B><FONT SIZE=+1>18-11573</FONT></B>
+        #       <B></B>Sara A Taylor
+        #     <BR>
+        #       <B>Case type:</B> bk
+        #       <B>Chapter:</B> 7
+        #       <B>Asset:</B> No
+        #       <B>Vol: </B> v
+        #       <b>Chief Judge:</b> Melvin S. Hoffman
+        #     <BR>
+        #       <B>Date filed:</B> 04/30/2018
+        #       <B>Date of last filing:</B> 06/06/2018
+        #     <BR>
+        #   </CENTER>
+
+        # Rather than take the approach used by DocketParser of
```

Comment:
I'm not sure it's worth distinguishing here that this is different than the docket parser. Why not just explain why this is good on its own? Feels like you're making a comment about *that* code over in this code.

Reply:
Well, it feels that way because I am. I'd like the reader to understand in context why the choice was made ,and pretending that code didn't influence this code would be...pretending.